.travis.yml

```diff
@@ -1,10 +1,21 @@
 sudo: false
-
 language: ruby
-
 rvm:
-  - 2.3.3
-  - 2.4.0
-
+- 2.4.5
 cache: bundler
 script: bundle exec middleman build
+deploy:
+  provider: s3
+  access_key_id: $AWS_ACCESS_KEY_ID
+  secret_access_key: $AWS_SECRET_ACCESS_KEY
+  bucket: developer.grooveapp.com
+  local-dir: build
+  upload-dir: documentation
```

Comment:
I'm uploading this stuff to a `documentation` dir and then the default behavior is to point to this directory.

Maybe this is cleaner than uploading all the files to the bucket directly?